it-IT/CecilesMonsterGiveItem_it_IT.lang

```diff
@@ -2,7 +2,7 @@
 language=Italiano
 //TODO Translate [Reference] '...?'
 staxel.village.dialogue.CecilesMonsterGiveItem.line:10000100=
-//TODO Translate [Reference] 'Here!'
+//TODO Translate [Reference] 'Ecco!'
```

Comment:
This is a comment showing the text in english 'Here!' in this case. you would put the italian on the next line like this:

staxel.village.dialogue.CecilesMonsterGiveItem.line:10000200=Ecco!

Reply:
Thank you for the correction, I will fix it now